stdlib/array.ml

```diff
@@ -30,6 +30,12 @@ external unsafe_blit :
 external create_float: int -> float array = "caml_make_float_vect"
 let make_float = create_float
 
+module Floatarray = struct
+  external create : int -> floatarray = "caml_floatarray_create"
+  external get : floatarray -> int -> float = "caml_floatarray_get"
```

Comment:
I know this is WIP, but do we agree that this should be a %-primitive instead?

Reply:
Yes of course.